### PR TITLE
FOUNDRYUI-103 & 122 Documentation for Installation, usage, and Readme updates

### PR DESCRIPTION
**Pull request by wmangimelli**
Updates to both the main and packages/hs-react-ui Readme.md files. Addition include:
Main:
- Foundry naming to replace Rasa
- Foundy Logo
- Badges for: Size, License, open source
- Technologies
HS-React-UI
- Getting Started/Installation
- Usage
- Badges for: Size, License, Open Source, Jest Coverage, Build passing
- Technologies
- Links to Composing example, GlobalStyling example, changelog
- Known Issue section

### CODE REVIEW

File: packages/hs-react-ui/README.md [hunk 1]
```diff
@@ -1,12 +1,192 @@
...
+import styled from 'styled-components';
+
+// Foundry provides a basic set of colors, but feel free to add your own
+const myColors = {
+  ...colors,
+  blue: 'blue',
+  };
+
+// We recommend starting with the Foundry provided styling, in this case 'Card.Container', 
+// and then add you own overrides or new styles.
```
Comment (joien1) [inline packages/hs-react-ui/README.md:128]:
```suggestion
// and then add your own overrides or new styles. 
```

File: packages/hs-react-ui/README.md [hunk 2]
```diff
@@ -63,3 +238,27 @@ const MyContainer = styled(Div)`styles here`;
...
+- Pushes to master trigger a release build
+- Auto versioning
+- Pushes to v1-release trigger a pre-release build
+build checks on every PR to master and v1-release
+- Change logs are generated (see link below)
+- storybook docs are generated and checked in for releases
+- Test coverage
+
+###What isn't working yet?
+- Publishing to NPM isn't working yet due to the authentication token not having proper permissions - we have a ticket submitted to get this fixed and Bennett has been doing an excellent job tackling those types of tickets for us :clapping:
```
Comment (donaldjbrady) [inline packages/hs-react-ui/README.md:253]: This part is working now and can be removed 😄

File: packages/hs-react-ui/README.md [hunk 3]
```diff
@@ -1,12 +1,192 @@
...
+##Usage
+
+###Hello World
+Get started with a simple hello world
+
+```
+import { Card } from '@headstorm/hs-react-ui'; 
+
+function HelloWorld() {
+    return <Card>Hello World!</Card>;
```
[the remark this answers is not in the document]
Comment (wmangimelli) [reply packages/hs-react-ui/README.md:72]: Great catch! Thank you, Jake!

File: packages/hs-react-ui/README.md (hunk 1 above)
Comment (wmangimelli) [reply packages/hs-react-ui/README.md]: Thank you, sir!

File: packages/hs-react-ui/README.md (hunk 2 above)
Comment (wmangimelli) [reply packages/hs-react-ui/README.md]: Nice. Update. Thanks, DJ

**Pull request merged**